Feat/xd 82

Sungonogi opened the pull request:
Pull request for the #82

Persists committed:
init: facilities

Signed-off-by: Jonas Heisterberg <jonas@mennicke.com>
Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
init: facilities

Signed-off-by: Jonas Heisterberg <jonas@mennicke.com>
Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
init: facilities

Signed-off-by: Jonas Heisterberg <jonas@mennicke.com>
Signed-off-by: Jonas <jonas@mennicke.com>

PatrickSchm1dt committed:
feat: added "about & leagal information" and short description on home page

Signed-off-by: Patrick Schmidt <patrick.m.schmidt@fau.de>
#111

Persists committed:
refactor: prisma Schema

Signed-off-by: Jonas Heisterberg <jonas@mennicke.com>
Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
refactor: prisma Schema

Signed-off-by: Jonas Heisterberg <jonas@mennicke.com>
Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
refactor: prisma Schema

Signed-off-by: Jonas Heisterberg <jonas@mennicke.com>
Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
refactor: update seeder :seedling:

Signed-off-by: Jonas Heisterberg <jonas@mennicke.com>
Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
refactor: update seeder :seedling:

Signed-off-by: Jonas Heisterberg <jonas@mennicke.com>
Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
refactor: update seeder :seedling:

Signed-off-by: Jonas Heisterberg <jonas@mennicke.com>
Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
refactor: update seeder :seedling:

Signed-off-by: Jonas Heisterberg <jonas@mennicke.com>
Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
refactor: update seeder :seedling:

Signed-off-by: Jonas Heisterberg <jonas@mennicke.com>
Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
refactor: update seeder :seedling:

Signed-off-by: Jonas Heisterberg <jonas@mennicke.com>
Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
refactor: rename entityId to assetId

Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
refactor: rename entityId to assetId

Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
refactor: rename entityId to assetId

Signed-off-by: Jonas <jonas@mennicke.com>

Hydraneut committed:
feat: 144: Set up case-brows-page, so that I can start development

Co-authored-by: Maximilian Krug <krugm03@gmail.com>
Signed-off-by: Alexander Lorenz <alex.lorenz.1@gmx.de>

Hydraneut committed:
feat: 144: Added to table, sort, color, higlighting, view open button

Signed-off-by: Alexander Lorenz <alex.lorenz.1@gmx.de>

Persists committed:
fix: time Series tests

Signed-off-by: Jonas Heisterberg <jonas@mennicke.com>
Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
fix: time Series tests

Signed-off-by: Jonas Heisterberg <jonas@mennicke.com>
Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
fix: time Series tests

Signed-off-by: Jonas Heisterberg <jonas@mennicke.com>
Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
fix: imports

Signed-off-by: Jonas Heisterberg <jonas@mennicke.com>
Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
fix: imports

Signed-off-by: Jonas Heisterberg <jonas@mennicke.com>
Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
fix: imports

Signed-off-by: Jonas Heisterberg <jonas@mennicke.com>
Signed-off-by: Jonas <jonas@mennicke.com>

Commit:
feat(frontend): added GET + POST case operatios
TODO: add DELETE + UPDATE, fix Date in POST
#104

Co-authored-by: Patrick Schmidt <patrick.m.schmidt@fau.de>
Signed-off-by: Sabo <saleb2k@gmail.com>

Commit:
fix(frontend): fixed dueDate in Case Create Page

Date is now properly read from the datepicker

Signed-off-by: Sabo <saleb2k@gmail.com>

#104

Commit:
feat(frontend): added Toast to Case Creation

fixed also columns in case overview page

Signed-off-by: Sabo <saleb2k@gmail.com>

#104

Persists committed:
feat: add facility seeder & aspect seeder

Signed-off-by: Jonas Heisterberg <jonas@mennicke.com>
Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
feat: add facility seeder & aspect seeder

Signed-off-by: Jonas Heisterberg <jonas@mennicke.com>
Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
feat: add facility seeder & aspect seeder

Signed-off-by: Jonas Heisterberg <jonas@mennicke.com>
Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
refactor: fix facilitie imports

Signed-off-by: Jonas Heisterberg <jonas@mennicke.com>
Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
refactor: fix facilitie imports

Signed-off-by: Jonas Heisterberg <jonas@mennicke.com>
Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
refactor: fix facilitie imports

Signed-off-by: Jonas Heisterberg <jonas@mennicke.com>
Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
feat: add comments for the facilities service

Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
feat: add comments for the facilities service

Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
feat: add comments for the facilities service

Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
fix: timeSeriesService imports

Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
fix: timeSeriesService imports

Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
fix: timeSeriesService imports

Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
fix: disable current seed script

Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
fix: disable current seed script

Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
fix: disable current seed script

Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
fix: adjust seeder

Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
fix: adjust seeder

Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
fix: adjust seeder

Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
feat: add dto to the single timeseries item endpoint

Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
feat: add dto to the single timeseries item endpoint

Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
feat: add dto to the single timeseries item endpoint

Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
refactor: adjust the timeseries tests

Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
refactor: adjust the timeseries tests

Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
refactor: adjust the timeseries tests

Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
fix: what is that spacing?

Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
fix: what is that spacing?

Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
fix: what is that spacing?

Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
feat: add facilities dto

Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
feat: add facilities dto

Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
feat: add facilities dto

Signed-off-by: Jonas <jonas@mennicke.com>

Commit:
feat(frontend): improved case detailspage and case list

replaced table in case list with event list

#104

Hydraneut committed:
feat: 144: Improved code-quality

Signed-off-by: Alexander Lorenz <alex.lorenz.1@gmx.de>

Commit:
fix(frontend): fixed date picker for case creation

previously it forwarded the current date instead of the picked one

Signed-off-by: Sabo <saleb2k@gmail.com>

#104

Hydraneut committed:
feat: 144: Changed colors to the siemens ones

Signed-off-by: Alexander Lorenz <alex.lorenz.1@gmx.de>

Hydraneut committed:
Merge branch 'develop' into feat/xd-144

Hydraneut committed:
feat: 144: fixing lints

Signed-off-by: Alexander Lorenz <alex.lorenz.1@gmx.de>

PatrickSchm1dt committed:
feat: removed console errors and fixed linting errors. Added accessor class for dueDate

Signed-off-by: Patrick Schmidt <patrick.m.schmidt@fau.de>
#104

Persists committed:
fix: timeseries without property set name

Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
fix: timeseries without property set name

Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
fix: timeseries without property set name

Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
fix: facilities assetId

Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
fix: facilities assetId

Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
fix: facilities assetId

Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
fix: facilities assetId

Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
fix: facilities assetId

Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
fix: facilities assetId

Signed-off-by: Jonas <jonas@mennicke.com>

Sungonogi commented:
I am not sure. These changes were all done by Jonas in the #110 branch. I just branched of feat/xd-110 because the timeseries stuff works there. Maybe I should have created the PR after the 110 branch is merged.

KonsumGandalf commented on libs/facilities/backend/facilities/README.md:
```diff
@@ -0,0 +1,7 @@
+# facilities-backend-facilities
+
+This library was generated with [Nx](https://nx.dev).
+
+## Running unit tests
+
+Run `nx test facilities-backend-facilities` to execute the unit tests via [Jest](https://jestjs.io).
```
docs

Persists replied:
Aye Aye sir

KonsumGandalf commented on libs/facilities/backend/timeseries/src/lib/dto/query.dto.ts:
```diff
@@ -47,5 +55,6 @@ export class GetTimeSeriesQueryDto implements IGetTimeseriesQuery {
 
 	@IsOptional()
 	@IsBoolean()
+	@Transform(({ value }) => value === 'true')
```
Think about implementing a custom validator for this like here the `ToBoolean` would greate easen later implementation.

https://stackoverflow.com/a/66697810

Persists replied:
I like ur idea, but since i think this query will later be removed i would just leave it like it is right now

KonsumGandalf commented on libs/facilities/backend/facilities/src/lib/service/faclities.service.ts:
```diff
@@ -0,0 +1,173 @@
+						data: aspects.map((aspect) => {
+							return {
+								propertySetName: aspect.name,
+								Variables: aspect.variables.map((variable) => {
```
Why is this in CamelCase? should be 
```suggestion
								variables: aspect.variables.map((variable) => {
```

Persists replied:
yes i fixed this

KonsumGandalf reviewed (changes requested):
LGTM good pr!

KonsumGandalf commented:
> I am not sure. These changes were all done by Jonas in the #110 branch. I just branched of feat/xd-110 because the timeseries stuff works there. Maybe I should have created the PR after the 110 branch is merged.

Did you merge or rebase? But yeah ur right theres a lot from jonas branch

Sungonogi commented:
No it is all from Jonas I did not change anything yet. I just created a PR because then I can see the changes and if tests run and everything (since this was not branched from develop but feat/xd-110)  better before I put my stuff there. Where I work we always create the PR and the feature branch at the same time and then start to work on it

Sungonogi commented:
Jonas told me that I should put the PR as draft in these situations so I put it to draft

PatrickSchm1dt committed:
refactor(frontend): fixed requests from review
#104

Co-authored-by: Saša Bojanic <saleb2k@gmail.com>
Signed-off-by: Patrick Schmidt <patrick.m.schmidt@fau.de>

PatrickSchm1dt committed:
Merge branch 'develop' into feat/xd-104

PatrickSchm1dt committed:
fix: fixed the lint errors

Signed-off-by: Patrick Schmidt <patrick.m.schmidt@fau.de>
#104

Hydraneut committed:
feat: 144: exported style and made the cursor indicate that cases are clickable

Signed-off-by: Alexander Lorenz <alex.lorenz.1@gmx.de>

PatrickSchm1dt committed:
refactor: removed the logo and added some small changes

Signed-off-by: Patrick Schmidt <patrick.m.schmidt@fau.de>
#111

PatrickSchm1dt committed:
Merge pull request #155 from amosproj/feat/xd-111

Feat/xd 111 Home Page and About & Legal Information

Persists committed:
refactor: prisma schema & enpoints

Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
refactor: prisma schema & enpoints

Signed-off-by: Jonas <jonas@mennicke.com>

Hydraneut committed:
Delete libs/cases/shared/models/src/lib/components directory

Hydraneut committed:
Update index.ts

Sungonogi committed:
refactor: small changes David told me to do
#82

Co-authored-by: David Schmidt <d.schmidt@campus.tu-berlin.de>
Signed-off-by: Ingo Sternberg <ingo.sternberg@fau.de>

Sungonogi committed:
refactor: small changes David told me to do
#82

Co-authored-by: David Schmidt <d.schmidt@campus.tu-berlin.de>
Signed-off-by: Ingo Sternberg <ingo.sternberg@fau.de>

Sungonogi committed:
refactor: small changes David told me to do
#82

Co-authored-by: David Schmidt <d.schmidt@campus.tu-berlin.de>
Signed-off-by: Ingo Sternberg <ingo.sternberg@fau.de>

Sungonogi committed:
refactor: small changes David told me to do
#82

Co-authored-by: David Schmidt <d.schmidt@campus.tu-berlin.de>
Signed-off-by: Ingo Sternberg <ingo.sternberg@fau.de>

Sungonogi committed:
feat/xd-82: small changes David helped me with (Code With Me)

Co-authored-by: David Schmidt <d.schmidt@campus.tu-berlin.de>
Signed-off-by: Ingo Sternberg <ingo.sternberg@fau.de>

Sungonogi committed:
feat/xd-82: small changes David helped me with (Code With Me)

Co-authored-by: David Schmidt <d.schmidt@campus.tu-berlin.de>
Signed-off-by: Ingo Sternberg <ingo.sternberg@fau.de>

Sungonogi committed:
feat/xd-82: small changes David helped me with (Code With Me)

Co-authored-by: David Schmidt <d.schmidt@campus.tu-berlin.de>
Signed-off-by: Ingo Sternberg <ingo.sternberg@fau.de>

Sungonogi committed:
feat/xd-82: small changes David helped me with (Code With Me)

Co-authored-by: David Schmidt <d.schmidt@campus.tu-berlin.de>
Signed-off-by: Ingo Sternberg <ingo.sternberg@fau.de>

PatrickSchm1dt committed:
Merge pull request #149 from amosproj/feat/xd-104

Feat/xd 104 - Case CRUD Operations

Persists committed:
refactor: add facilitiees interface

Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
refactor: add facilitiees interface

Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
refactor: add interface to service

Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
refactor: add interface to service

Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
refactor: remove type cast on facility response

Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
refactor: remove type cast on facility response

Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
refactor: add typing for  facilities rsponse

Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
refactor: add typing for  facilities rsponse

Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
refactor: date response for createdAt & updatedAt in facilities

Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
refactor: date response for createdAt & updatedAt in facilities

Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
feat: add first controller test

Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
feat: add first controller test

Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
feat: finish facilities tests

Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
feat: finish facilities tests

Signed-off-by: Jonas <jonas@mennicke.com>

shahraz1998 committed:
Upload deliverables

Signed-off-by: Shahraz Nasir <shahraz@ag-prop.com>

Persists committed:
Merge pull request #162 from amosproj/docu/sprint-7

Upload deliverables

Sungonogi committed:
feat: actually take facilities from the api/facilities endpoint

Signed-off-by: Ingo Sternberg <ingo.sternberg@fau.de>
#82

Sungonogi committed:
feat: actually take facilities from the api/facilities endpoint

Signed-off-by: Ingo Sternberg <ingo.sternberg@fau.de>
#82

Sungonogi committed:
Merge remote-tracking branch 'origin/feat/xd-82' into feat/xd-82

# Conflicts:
#	libs/facilities/backend/facilities/src/lib/controller/facilities.controller.ts
#	libs/facilities/backend/facilities/src/lib/facilities.module.ts
#	libs/facilities/backend/facilities/src/lib/service/faclities.service.ts
#	libs/facilities/frontend/domain/src/lib/application/facades/browse.facade.spec.ts
#	libs/facilities/frontend/domain/src/lib/application/facades/browse.facade.ts
#	libs/facilities/frontend/view/src/lib/components/detail/detail.page.ts
#	prisma/schema.prisma
#	prisma/seed.ts

Sungonogi commented:
because I had to branch of feat/xd-110 and then had to get an insane amount of changes from the develop branch again this PR looks weird. The actual change I worked on can be seen in this commit:

https://github.com/amosproj/amos2024ss01-xcelerator-demo-app/pull/152/commits/47c67edd793eea48985da0778086c70ebf93efcf

Hydraneut committed:
feat: 144: mock cases are now generated by faker, ngClass function is now part of .ts

Signed-off-by: Alexander Lorenz <alex.lorenz.1@gmx.de>

HaruspexSan committed:
Update README.md

Created the first part of the read me file, this consists of the first view points: participants, short summary of the project, link to Architecture document

Hydraneut committed:
Merge branch 'develop' into feat/xd-144

Hydraneut committed:
feat: 144: fixing lint

Signed-off-by: Alexander Lorenz <alex.lorenz.1@gmx.de>

Hydraneut committed:
Merge pull request #165 from amosproj/HaruspexSan-patch-4

Update README.md

HaruspexSan committed:
bug: 166 change icon size from 32 to default

Signed-off-by: Maximilian Krug <krugm03@gmail.com>
#166

Hydraneut committed:
Merge pull request #167 from amosproj/bug/xd-166

bug: 166 change icon size from 32 to default

Sungonogi committed:
rebase: get Jonas changes from feat/xd-110 because I didn't rebase/merge correctly
#82

Hydraneut committed:
feat: 144: removing cluter

Signed-off-by: Alexander Lorenz <alex.lorenz.1@gmx.de>

Sungonogi committed:
chore: rename the locked variable as requested by David

Signed-off-by: Ingo Sternberg <ingo.sternberg@fau.de>
#82

Sungonogi committed:
chore: rename the locked variable as requested by David

Signed-off-by: Ingo Sternberg <ingo.sternberg@fau.de>
#82

Sungonogi committed:
chore: change the test as requested by David

Signed-off-by: Ingo Sternberg <ingo.sternberg@fau.de>
#82

Sungonogi committed:
chore: change the test as requested by David

Signed-off-by: Ingo Sternberg <ingo.sternberg@fau.de>
#82

Sungonogi committed:
feat/xd-82: change timeseries request service to fit the details pages needs

Signed-off-by: Ingo Sternberg <ingo.sternberg@fau.de>
#82

Sungonogi committed:
feat/xd-82: change timeseries request service to fit the details pages needs

Signed-off-by: Ingo Sternberg <ingo.sternberg@fau.de>
#82

Sungonogi committed:
feat/xd-82: get chart data from the backend, delete chart componen and facility.mocks/charts

Signed-off-by: Ingo Sternberg <ingo.sternberg@fau.de>
#82

Sungonogi committed:
feat/xd-82: get chart data from the backend, delete chart componen and facility.mocks/charts

Signed-off-by: Ingo Sternberg <ingo.sternberg@fau.de>
#82

Persists committed:
refactor: _time to time

Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
refactor: add location to facilities

Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
fix: adjust error Handling

Persists committed:
fix: iot hub test

Commit:
docs:readme

HaruspexSan committed:
Merge pull request #169 from sn0wcat/feat/update-readme

docs:readme

PatrickSchm1dt committed:
fix: fixed the height of the welcome message

PatrickSchm1dt committed:
Merge pull request #170 from amosproj/bug/xd-111

fix: fixed the height of the welcome message

Hydraneut committed:
Merge pull request #151 from amosproj/feat/xd-144

Feat/xd 144

HaruspexSan committed:
Create dummy

ceciliabetb committed:
Add files via upload

ceciliabetb committed:
Add files via upload

HaruspexSan committed:
Merge pull request #173 from amosproj/ceciliabetb-patch-6

Add files via upload

ceciliabetb committed:
Add files via upload

HaruspexSan committed:
Merge pull request #175 from amosproj/ceciliabetb-patch-8

Add files via upload

HaruspexSan committed:
Merge pull request #174 from amosproj/ceciliabetb-patch-7

Add files via upload

Sungonogi committed:
get Jonas changes from branch feat/xd-110
#82

Sungonogi committed:
feat/xd-82: actually use item.time now that Jonas fixed it
#82

Sungonogi committed:
feat/xd-82: actually use item.time now that Jonas fixed it
#82

HaruspexSan committed:
Merge pull request #172 from amosproj/HaruspexSan-patch-5

Create dummy

HaruspexSan committed:
Add files via upload

HaruspexSan committed:
Merge pull request #176 from amosproj/HaruspexSan-patch-6

Add files via upload

Sungonogi committed:
feat/xd-82: make charts look nice
#82

Sungonogi committed:
feat/xd-82: make charts look nice
#82

HaruspexSan committed:
Update README.md

Import the pictures of the current state of the website

Sungonogi committed:
feat/xd-82: I forgot to add the title, note: I forgot to sign off the last commits. I basically did everything on the details page (the charts)

Signed-off-by: Ingo Sternberg <ingo.sternberg@fau.de>
#82

Sungonogi committed:
feat/xd-82: I forgot to add the title, note: I forgot to sign off the last commits. I basically did everything on the details page (the charts)

Signed-off-by: Ingo Sternberg <ingo.sternberg@fau.de>
#82

HaruspexSan committed:
Merge pull request #177 from amosproj/HaruspexSan-patch-7

Update README.md

Persists committed:
feat: add caching of api calls

Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
feat: add some facility tests

Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
fix: imports

Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
fix: seed script

Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
fix: lint

Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
fix: lint

Signed-off-by: Jonas <jonas@mennicke.com>

Persists committed:
fix: comments

Co-authored-by: David Schmidt <d.schmidt@campus.tu-berlin.de>

KonsumGandalf committed:
Merge pull request #168 from amosproj/feat/xd-110

Feat/xd 110

KonsumGandalf committed:
ci: github actions

Prevent husky from run prepare script.

KonsumGandalf committed:
Merge pull request #178 from amosproj/fix/docker-build

ci: github actions

KonsumGandalf committed:
ci(backend): skip husky install

KonsumGandalf committed:
feat(cases-shared-models): move case to shared folfer

KonsumGandalf committed:
build: allow to import cases

KonsumGandalf committed:
Merge pull request #179 from amosproj/fix/docker-build

fix: docker build

Sungonogi committed:
feat/xd-82: get changes from develop

Signed-off-by: Ingo Sternberg <ingo.sternberg@fau.de>
#82

Sungonogi committed:
feat/xd-82: get changes from develop

Signed-off-by: Ingo Sternberg <ingo.sternberg@fau.de>
#82

Sungonogi committed:
Merge remote-tracking branch 'origin/feat/xd-82' into feat/xd-82
